src/Lean/Elab/Tactic/BVDecide/LRAT/Actions.lean

```diff
@@ -0,0 +1,44 @@
+/-
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Josh Clune
+-/
+prelude
+import Std.Sat.CNF
+
+/-!
+This module contains the definition of the LRAT format as a type `Action`, that is polymorphic over
```

Comment:
Since this file is where the LRAT format is defined I think it makes sense to link to some resource on the topic here for people who are curious. Something like https://www.cs.utexas.edu/~marijn/publications/lrat.pdf or whatever exposition you think is good and matches what we're doing here.